src/main/java/ua/danit/final_project/entities/Schedule.java

```diff
@@ -2,6 +2,8 @@
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'com.fasterxml.jackson.annotation.JsonIgnoreProperties'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=83&issues=AWVY-faI2xJdnw6QkSxi&open=AWVY-faI2xJdnw6QkSxi)

[ISSUE_KEY]: AWVY-faI2xJdnw6QkSxi